### PR TITLE
fix(*): fault injection protocols

### PR DESCRIPTION
**Pull request by lobkovilya**
Signed-off-by: Ilya Lobkov <ilya.lobkov@konghq.com>

### CODE REVIEW

File: docs/docs/1.1.x/policies/fault-injection.md [hunk 1]
```diff
@@ -65,7 +65,9 @@ spec:
 `FaultInjection` is a policy, which is applied to the connection between dataplanes. As most of the policies, `FaultInjection` supports the powerful mechanism of matching, which allows you to precisely match source and destination dataplanes.
 
 ::: warning
-`FaultInjection` policy available only for L7 HTTP traffic, `kuma.io/protocol: http` is mandatory tag for the destination selector.
+`FaultInjection` policy available only for L7 HTTP traffic,
```
Comment (bartsmykla) [inline docs/docs/1.1.x/policies/fault-injection.md:68]: shouldn't be
```suggestion
`FaultInjection` policy is available only for L7 HTTP traffic, 
```
?

File: docs/docs/1.1.x/policies/fault-injection.md [hunk 2]
```diff
@@ -65,7 +65,9 @@ spec:
 `FaultInjection` is a policy, which is applied to the connection between dataplanes. As most of the policies, `FaultInjection` supports the powerful mechanism of matching, which allows you to precisely match source and destination dataplanes.
 
 ::: warning
-`FaultInjection` policy available only for L7 HTTP traffic, `kuma.io/protocol: http` is mandatory tag for the destination selector.
+`FaultInjection` policy available only for L7 HTTP traffic, 
+`kuma.io/protocol` is mandatory tag for the destination selector.
```
Comment (lahabana) [inline docs/docs/1.1.x/policies/fault-injection.md:69]:
```suggestion
therefore `kuma.io/protocol` is a mandatory tag for the destination selector and must be of value `http`, `http2` or `grpc`.
```

File: docs/docs/1.1.x/policies/fault-injection.md (hunk 1 above)
Comment (lobkovilya) [reply docs/docs/1.1.x/policies/fault-injection.md:68]: Right, thanks! Let me backport this for all versions

File: docs/docs/1.1.x/policies/fault-injection.md [hunk 3]
```diff
@@ -65,7 +65,9 @@ spec:
...
+Allowed values are `http`, `http2` and `grpc`.
```
Comment (lahabana) [inline docs/docs/1.1.x/policies/fault-injection.md:70]:
```suggestion
```

**Comment by lahabana** [comment]
Need to do like `/format` in Kuma but with backport.sh

**Comment by lahabana** [comment]
@lobkovilya if you rebase on master checklink will pass again

**Pull request merged**